### ISSUE TITLE
Can't Install Starter Kit locally

### ISSUE THREAD

**Issue by lokmanm** [statamic/cli#77]
This used to work but now when I'm trying to run:

```
statamic new --with-config --local landtamic lucky-media/landtamic 
```

And yes I do have the global config updated to:

```json
{
	"repositories": [
		{
			"type": "path",
			"url": "/Users/Sites/landtamic"
		}
	]
}
```

When I run the command it still prompts me for a license.

**Comment by lokmanm** [comment]
Now I get: `Starter kit is required when using `--local` option!`

**Comment by lokmanm** [comment]
@robdekort yep I have all that. I think its failing because of the license check. It works for me for our free starter kit, but fails for the paid ones.

**Comment by robdekort** [comment]
Ehmmm, ok, no experience there I'm afraid.
